apps/admin/frontend/src/screens/tally/confirm_remove_cvrs_modal.tsx

```diff
@@ -0,0 +1,119 @@
+import { Modal, P, Button } from '@votingworks/ui';
+import React from 'react';
+import {
+  getManualResultsMetadata,
+  getCastVoteRecordFiles,
+  getCastVoteRecordFileMode,
+  clearCastVoteRecordFiles,
+  deleteAllManualResults,
+} from '../../api';
+
+export function ConfirmRemoveCvrsModal({
+  onClose,
+}: {
+  onClose: VoidFunction;
+}): JSX.Element | null {
+  const manualDataResultsMetadataQuery = getManualResultsMetadata.useQuery();
+  const castVoteRecordFilesQuery = getCastVoteRecordFiles.useQuery();
+  const castVoteRecordFileModeQuery = getCastVoteRecordFileMode.useQuery();
+  const clearCastVoteRecordFilesMutation =
+    clearCastVoteRecordFiles.useMutation();
+  const deleteAllManualResultsMutation = deleteAllManualResults.useMutation();
+
+  if (
+    !castVoteRecordFilesQuery.isSuccess ||
+    !manualDataResultsMetadataQuery.isSuccess ||
+    !castVoteRecordFileModeQuery.isSuccess
+  ) {
+    return null;
+  }
+
+  const isTestMode = castVoteRecordFileModeQuery.data === 'test';
+  const hasManualData = manualDataResultsMetadataQuery.data.length > 0;
+
+  function removeCvrs() {
+    clearCastVoteRecordFilesMutation.mutate(undefined, {
+      onSuccess: onClose,
+    });
+  }
+
+  async function removeCvrsAndManualResults() {
+    await Promise.all([
+      clearCastVoteRecordFilesMutation.mutateAsync(),
+      deleteAllManualResultsMutation.mutateAsync(),
```

Comment:
How do you feel about combining these to make a new mutation endpoint? I don't really like the idea of one failing and the other succeeding, though I _think_ that'd be okay here if a bit confusing.

Reply:
Hmm, good question. I like that idea.

I peeked at the backend and there would also need to be some work done to manage the transactions. We're already not great about that (e.g. `clearCastVoteRecords` does two different db writes in separate transactions). 

Given this PR is focused on frontend changes, I think it's probably out of scope. But maybe there's a larger tech improvement to make in terms of transaction management?